lib/react-permissible.js

```diff
@@ -1 +1,6 @@
+object-assign
+(c) Sindre Sorhus
```

Comment:
why Sindre Sorhus?

Reply:
Not sure to be honest. Probably some external library is "copyrighted" by @sindresorhus 😉